ldm/modules/encoders/modules.py

```diff
@@ -150,14 +149,14 @@ class BERTEmbedder(AbstractEncoder):
     """Uses the BERT tokenizr model and add some transformer encoder layers"""
 
     def __init__(
-        self,
-        n_embed,
-        n_layer,
-        vocab_size=30522,
-        max_seq_len=77,
-        device=choose_torch_device(),
-        use_tokenizer=True,
-        embedding_dropout=0.0,
+            self,
```

Comment:
Is the extra indentation here intentional?

Reply:
The original code had odd indentation here. and I just tidied it up. It's in the method call signature and I don't think changes the semantics.